### PR TITLE
Add test HTML file

### PR DESCRIPTION
**Pull request by agerard57**
# Context

**_Hello world !_**

I've made an HTML file which, to be honest, doesn't serve any purposes whatsoever.
However, @nicolaskempf57, you can use this PR if you intend to show how this feature works ;)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<html>
+    <body>
```
Comment (agerard57) [inline index.html:2]: Forgot to add an header / head

File: index.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+        <h1>Hello World<h1>
```
Comment (nicolaskempf57) [inline index.html:3]: Nice h1

**Review by nicolaskempf57 — APPROVED** [review]

**Review by agerard57 — COMMENTED** [review]
I should have added an head tag

File: index.html (hunk 2 above)
Comment (agerard57) [reply index.html:3]: Thanks !

**Pull request merged**